src/window.h

```diff
@@ -296,6 +310,14 @@ CORE_LIBSPEC void meter_interior(unsigned int progress, unsigned int out_of);
 #define DI_DEBUG_LABEL        DI_INPUT_BOX_LABEL
 #define DI_DEBUG_NUMBER       79
 
+#define DI_SLOTSEL_NORMAL     7
+#define DI_SLOTSEL_HIGHLIGHT  11
```

Comment:
For whatever reason the highlight color looks way too close to the normal color for me (moreso on the tiny NDS screen than on PC). I haven't tried any alternatives yet on my NDS but normal=8 and highlight=10 looked promising. Color 8 might be way too hard to read on some displays though.

Reply:
Resolved in 49010f1.